components/desktop/gnome2/planner/Makefile

```diff
@@ -25,11 +26,11 @@ COMPONENT_ARCHIVE_HASH=	\
     sha256:130d29e6d6b8c1994017deba2acca05a4bf83ba1e169635ec8c5c7878abe0696
 COMPONENT_ARCHIVE_URL=	http://ftp.gnome.org/pub/GNOME/sources/$(COMPONENT_NAME)/0.14/$(COMPONENT_ARCHIVE)
 
-include $(WS_TOP)/make-rules/prep.mk
-include $(WS_TOP)/make-rules/configure.mk
-include $(WS_TOP)/make-rules/ips.mk
+include $(WS_MAKE_RULES)/prep.mk
+include $(WS_MAKE_RULES)/configure.mk
+include $(WS_MAKE_RULES)/ips.mk
 
-PATH=/usr/gnu/bin:/usr/bin:/usr/perl5/bin
+PATH = $(PATH.gnu)
 
 COMPONENT_PREP_ACTION =        ( cd $(@D) && \
 					aclocal -I. && \
```

Comment:
I found the problem with libchamplain: Some library files were installed in the BE without a package manager.  This would have happened when I built the JDS consolidation in that BE.

I'm going to omit all three of these components from this batch of changes.  I have several other batches to submit once this one is completed.